stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -175,7 +175,12 @@ public class ChatClient internal constructor(
     private var pushNotificationReceivedListener: PushNotificationReceivedListener =
         PushNotificationReceivedListener { _, _ -> }
 
-    public lateinit var plugins: List<Plugin>
+    /**
+     * The list of plugins added once user is connected.
+     *
+     * @see [Plugin]
+     */
+    internal var plugins: List<Plugin> = emptyList()
```

Comment:
Do we need to keep it public? 🤔

Reply:
I guess for testing it will be useful. Otherwise, we need to add a factory, etc for testing.

Reply:
👍 we can always make it public and annotate with `@InternalStreamChatApi`, but should we expose it to users? 🤔

Reply:
No for users. Doesn't make sense when we provide a factory to add plugins for the users.